frontend/src/utilities/quickStartUtils.ts

```diff
@@ -26,7 +26,7 @@ export const getLaunchStatus = (
   }
 
   const quickStartState = qsContext.allQuickStartStates[quickStartId];
-
+  //eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
```

Comment:
disabling the type because the type comes from the `@patternfly/quickstarts`

Reply:
Here's a sneaky fix. Update this line and the `if` statement above to:
```
  if (!quickStartId || !qsContext) {
    return LaunchStatusEnum.Open;
  }

  const quickStartState = qsContext.allQuickStartStates?.[quickStartId];
```

Reply:
I tried that , but its giving me  `Unnecessary optional chain on a non-nullish value`

Reply:
@pnaik1 did you remove `|| !qsContext.allQuickStartState` from the `if` statement above the error?